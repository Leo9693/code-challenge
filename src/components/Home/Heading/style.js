import styled, { keyframes } from "styled-components";
import { zoomIn } from "react-animations";
export const Heading = styled.div`
    display: flex;
    justify-content: space-between;
    align-items: center;
    font-size: 2rem;
    color: white;
    padding: 2rem 0;
    transition: all 1s;
    animation: ${keyframes`${zoomIn}`} 0.6s ease-in-out;
    img {
        max-width: 200px;
        display: block;
        &:hover {
            cursor: pointer;
        }
    }
    @media (max-width: 768px) {
        flex-direction: column;
        font-size: 1.3rem;
        & > h4 {
            margin-top: 1rem;
        }
    }
    @media (max-width: 576px) {
        flex-direction: column;
        font-size: 1.1rem;
    }
`;
