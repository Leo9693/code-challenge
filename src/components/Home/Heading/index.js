import React from "react";
import logo from "../../../img/qantas.svg";
import { Heading } from "./style";
import { Link } from "react-router-dom";
function index() {
    return (
        <Heading>
            <Link to="/">
                <img src={logo} alt="Logo" />
            </Link>
            <h3>Welcome to Quatas Online Service</h3>
        </Heading>
    );
}

export default index;
