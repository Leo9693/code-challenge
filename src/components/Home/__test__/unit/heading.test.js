import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Heading from "../../Heading";
Enzyme.configure({ adapter: new Adapter() });

test("test ariport Types component", () => {
    const wrapper = shallow(<Heading />);
    expect(wrapper.debug()).toMatchSnapshot();
});
