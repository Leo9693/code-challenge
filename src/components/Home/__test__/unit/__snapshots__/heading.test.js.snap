// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ariport Types component 1`] = `
"<styled.div>
  <Link to=\\"/\\" className=\\"d-flex\\">
    <img src=\\"qantas.svg\\" alt=\\"Logo\\" />
  </Link>
  <h3>
    Welcome to Quatas Online Service
  </h3>
</styled.div>"
`;
